### PR TITLE
Add release docs for 0.7

### PR DESCRIPTION
**Pull request by torbsto**

### CODE REVIEW

File: docs/docs/roadmap.md [hunk 1]
```diff
@@ -6,22 +6,17 @@ and [project boards](https://github.com/bakdata/quick/projects).
...
 Development: Q3 2022
 
 * Improved gateway performance: Pre-computation of key's location
-* Kafka-Streams library for custom REST APIs in Quick
+* Kafka 3.0 support
 
 ### Further ideas
 
+* Kafka-Streams library for custom REST APIs in Quick
+* More complex query arguments, like ranges and filter
```
Comment (DawidNiezgodka) [inline docs/docs/roadmap.md:19]: Only a single filter? Or ranges and filter**s**?

File: docs/docs/roadmap.md [hunk 2]
```diff
@@ -6,22 +6,17 @@ and [project boards](https://github.com/bakdata/quick/projects).
...
-Development: Q2 2022
-
-* Protobuf support
-* Kafka 3.0 support
-
 ### 0.8
 
 Development: Q3 2022
 
 * Improved gateway performance: Pre-computation of key's location
```
Comment (DawidNiezgodka) [inline docs/docs/roadmap.md:13]: This was already here but shouldn't it be either of a key's location or keys' location depending on whether singular or plural is meant.

File: docs/docs/roadmap.md (hunk 1 above)
Comment (torbsto) [reply docs/docs/roadmap.md:19]: - ranges -> two arguments, e.g. from, to
- filter -> one argument, e.g. smaller than

Comment (DawidNiezgodka) [reply docs/docs/roadmap.md:19]: That's really a trifle - if filter is countable and we have a single one, then we might write "a filter" and not just filter.

Comment (torbsto) [reply docs/docs/roadmap.md]: filters it is then :smile:

**Pull request merged**